src/chrome/chromeDebugAdapter.ts

```diff
@@ -2729,7 +2729,10 @@ export abstract class ChromeDebugAdapter implements IDebugAdapter {
             } else {
                 propCountP = this.getCollectionNumPropsByEval(object.objectId);
             }
-        } else {
+        } else if (context === 'watch' && object.subtype && object.subtype === 'error') {
+            propCountP = Promise.reject(new Error(errors.evalNotAvailableMsg));
```

Comment:
This doesn't seem like the right place for this fix. This error normally comes from https://github.com/Microsoft/vscode-chrome-debug-core/blob/master/src/chrome/chromeDebugAdapter.ts#L2492. Why does that not work for Edge? I can't repro this bug on Chrome.

Reply:
I'm sorry, this is my mistake. The reason non-existent watch variables in edge don't get caught is because edge does not have the property exceptionDetails when it comes back from the browser. It just returned with results as an empty string. I should have checked that it was edge specific, my bad. I'm making the fix on the edge debug adapter now.